DSL/Resql/training/add-intent.sql

```diff
@@ -1,6 +1,2 @@
 INSERT INTO intent (intent, created, status)
-VALUES (:intent, CURRENT_TIMESTAMP, :status)
-ON CONFLICT (id) DO UPDATE
```

Comment:
@1AhmedYasser, The `id` is an auto-incremental sequence, therefore, this conflict should never have happened, right?

Reply:
Logically, yes